tools/hdf_dump_translation_dataset.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3
+
+"""
+This script creates the same HDF dataset as hdf_dump.py called on a TranslationDataset, but does so much faster
+and using much less memory.
+
+Input parameters are basically the files in a TranslationDataset folder. Additionally, the number of lines has to
+be given.
+"""
+
+import sys
+import argparse
+import gzip
+import pickle
+import numpy
+import h5py
+from operator import itemgetter
+from itertools import islice, zip_longest
+
+UNKOWN_LABEL = "<UNK>"
+POSTFIX = " </S>"
+
+
+class HDFTranslationDatasetCreator(object):
+  def __init__(self, hdf_file, source_file, target_file, source_vocabulary, target_vocabulary, number_of_lines,
```

Comment:
The arguments (including types) are not documented. (Please enable the warning about missing argument documentation in PyCharm to get notified about this.)